project/Dependencies.scala

```diff
@@ -0,0 +1,36 @@
+import sbt._
+
+object Dependencies {
+  val akkaVersion = System.getProperty("override.akka.version", "2.5.23")
```

Comment:
Would be neat to test 2 akka versions consecutively (eventually) with the proto 2, proto3

Reply:
We do, this is overridden in the test between deployments